ftdetect/slim.vim

```diff
@@ -1 +1 @@
-autocmd BufNewFile,BufRead *.slim setf slim
+autocmd BufNewFile,BufRead *.slim setfiletype slim
```

Comment:
I can't tell if you are trolling or not :D
The idea is to stick with `setf[ilename]` by default, as it (!) does not overrides filetype if it was set. If we will switch to `set filetype=slim` then user won't be able to "change" our default behavior easily.